Common subdirectories: cpufrequtils-008/bench and cpufrequtils-008.new/bench
Common subdirectories: cpufrequtils-008/build and cpufrequtils-008.new/build
Only in cpufrequtils-008.new/: cpufreq-aperf
Only in cpufrequtils-008.new/: cpufreq-info
Only in cpufrequtils-008.new/: cpufreq-set
Common subdirectories: cpufrequtils-008/debug and cpufrequtils-008.new/debug
Common subdirectories: cpufrequtils-008/lib and cpufrequtils-008.new/lib
Only in cpufrequtils-008.new/: libcpufreq.so
Only in cpufrequtils-008.new/: libcpufreq.so.0
Only in cpufrequtils-008.new/: libcpufreq.so.0.0.0
diff -u cpufrequtils-008/Makefile cpufrequtils-008.new/Makefile
--- cpufrequtils-008/Makefile	2012-05-06 01:17:18.000000000 +0200
+++ cpufrequtils-008.new/Makefile	2013-08-16 20:52:29.961086536 +0200
@@ -205,7 +205,8 @@
 	@xgettext --default-domain=$(PACKAGE) --add-comments \
 		--keyword=_ --keyword=N_ $(UTIL_SRC) && \
 	test -f $(PACKAGE).po && \
-	mv -f $(PACKAGE).po po/$(PACKAGE).pot
+	mv -f $(PACKAGE).po po/$(PACKAGE).pot && \
+        sed -i -e'/POT-Creation/d' po/*.pot
 
 update-gmo: po/$(PACKAGE).pot
 	 @for HLANG in $(LANGUAGES); do \
@@ -217,6 +218,7 @@
 			echo "msgmerge for $$HLANG failed!"; \
 			rm -f po/$$HLANG.new.po; \
 		fi; \
+		sed -i -e'/POT-Creation/d' po/*.po; \
 		msgfmt --statistics -o po/$$HLANG.gmo po/$$HLANG.po; \
 	done;
 
Common subdirectories: cpufrequtils-008/man and cpufrequtils-008.new/man
Common subdirectories: cpufrequtils-008/po and cpufrequtils-008.new/po
Common subdirectories: cpufrequtils-008/utils and cpufrequtils-008.new/utils
